xap-enterprise/Dockerfile

```diff
@@ -1,31 +1,31 @@
-ARG JAVA_IMAGE=openjdk
-ARG JAVA_TAG=8-alpine
-FROM ${JAVA_IMAGE}:${JAVA_TAG}
+ARG BASE_IMAGE=registry.access.redhat.com/ubi8/ubi-minimal
+ARG BASE_TAG=8.4
+FROM ${BASE_IMAGE}:${BASE_TAG}
 ARG GS_PRODUCT=xap
 ARG GS_VERSION=16.0.0
 ARG GS_BUILD_NAME=16.0.0-m21-sun-8
 ARG GS_NAME=gigaspaces-${GS_PRODUCT}-enterprise-${GS_BUILD_NAME}
 ARG GS_URL=https://gigaspaces-releases-eu.s3.amazonaws.com/xap/${GS_VERSION}/${GS_NAME}.zip
 ENV GS_HOME /opt/gigaspaces
 ENV GS_PRODUCT ${GS_PRODUCT}
+ENV KUBERNETES_CLINET_URL=https://resources.gigaspaces.com/k8s-java-client/k8s-java-client.jar
```

Comment:
I'm not a fan of having a dependency on our urls. I'm checking alternatives.
If we do stay with this approach, I think the artifact should be versioned (-5.0.0) for forward compatibility)

Reply:
I've tried a different approach but ended up with additional complication, so we can stick with this one.
Still, I think we should have a versioned jar to avoid compatibility issues when we change the k8s client version.

Reply:
I'll just add the version number to the path in S3 for the moment.

Reply:
I took care of that.